profiles/serializers.py

```diff
@@ -16,15 +14,14 @@ class Meta:
         fields = "__all__"
 
 
-class ProfileSerializer(serializers.ModelSerializer):
-    activity = ActivitySerializer(many=True, read_only=True)
-    category = CategorySerializer(many=True, read_only=True)
-    person = serializers.PrimaryKeyRelatedField(queryset=CustomUser.objects.all())
+class ProfileListSerializer(serializers.ModelSerializer):
     is_saved = serializers.SerializerMethodField()
 
     class Meta:
         model = Profile
-        fields = "__all__"
+        fields = ('name', 'person', 'is_registered', 'is_startup', 'official_name', 'region', 'common_info', 'address',
```

Comment:
Alter returned `activities` and `categories` fields so that they contain their names. Right now we get just lists of numbers (IDs) accordingly and so we'll have to make extra requests on the frontend to simply get those names of activities and categories.

Reply:
fixed)